### DISCUSSION TITLE
Flutter Local Notification Sound not working

### DISCUSSION THREAD

**Discussion by Temp-insta** [community/community#20517] [Projects and Issues] [answered]
The notification sound using Flutter Local Notification for Android not working at all here is the code
```sh
const NotificationDetails(
          android: AndroidNotificationDetails(
            'daily notification channel id',
            'daily notification channel name',
            'daily notification description',
            playSound: true,
            sound: RawResourceAndroidNotificationSound('pop'),
            importance: Importance.max,
            priority: Priority.high,
          ),
        ),

```
And I have pop.mp3 in this path: \android\app\src\main\res\raw\pop.mp3

How can I play the sound?

**Comment by santosharron** [comment, marked as answer]
The issue is that Xiaomi has specific issues and, by default, disallows notification sound, which I discovered while testing the app on Xiaomi. It somehow failed to function on the emulator as well, but after trying it on other actual devices, it did.

**Reply by sunilsinghchaudhary00** [reply]
> The issue is that Xiaomi has specific issues and, by default, disallows notification sound, which I discovered while testing the app on Xiaomi. It somehow failed to function on the emulator as well, but after trying it on other actual devices, it did.

but the question is how to overcome this problem , how to force xiomi phone also play custom sound any idea

**Reply by Blacktaler** [reply]
but the other apps' notifications have sound on Xiaomi. Only flutter apps' notification doesn't play sound
